cartoframes/data/observatory/enrichment/enrichment.py

```diff
@@ -137,8 +135,11 @@ def enrich_polygons(self, dataframe, variables, geom_column='geometry', filters=
         Data Observatory.
 
         When a polygon intersects with multiple geographies, the proportional part of the intersection will be used
-        to interpolate the quantity of the polygon value intersected, aggregating them
-        with the operator provided by `agg_operators` argument. See :obj:`VariableAggregation` for more info.
+        to interpolate the quantity of the polygon value intersected, aggregating them. Most of :obj:`Variable`
+        instances have a :py:attr:`Variable.agg_method` property what is used by default as aggregation function, but
+        you can overwrite it using the `aggregation` parameter (not even doing the aggregation). If a variable does not
+        have the `agg_method` property set and you do not overwrite it either (with the `aggregation` parameter), the
+        variable column will be skipped from the enrichment.
 
         Args:
             dataframe (pandas `DataFrame`, geopandas `GeoDataFrame`
```

Comment:
Remove it, a pandas `DataFrame` should not be accepted

Reply:
A DataFrame is allowed if it contains a column with a geometry column (in any supported encoding) and you specify the "geom_column" param.

Reply:
I rather prefer to limit to GeoDataFrame. Check this issue: https://github.com/CartoDB/cartoframes/issues/1255

We should give the user the message, any time you provide a geometry you need to provide it like an instance of GeoDataFrame. If not, we need to do things like give the geom column as a parameter.

Reply:
At isochrones we're accepting a GeoDataFrame but not giving the option of the `geom_column`. I think we should remove it

Reply:
In any case, @oleurud lets keep the `DataFrame` in this PR since we've the other issue for that #1255